src/plugins/controls/public/control_group/editor/edit_control_group.tsx

```diff
@@ -49,15 +48,14 @@ export const EditControlGroup = ({
 
     const flyoutInstance = openFlyout(
       toMountPoint(
-        <PresentationUtilProvider>
-          <ControlGroupEditor
-            initialInput={controlGroupContainer.getInput()}
-            updateInput={(changes) => controlGroupContainer.updateInput(changes)}
-            controlCount={Object.keys(controlGroupContainer.getInput().panels ?? {}).length}
-            onDeleteAll={() => onDeleteAll(flyoutInstance)}
-            onClose={() => flyoutInstance.close()}
-          />
-        </PresentationUtilProvider>
```

Comment:
The `ControlGroupEditor` doesn't need to be wrapped in the `PresentationUtilProvider` because it doesn't actually use any of the provided services.